salt/reactors/edx/pulsar_mitx.sls

```diff
@@ -1,8 +1,9 @@
 pulsar_mitx:
   local.slack.post_message:
-    - tgt: {{ data['id'] }}
+    - tgt: 'roles:master'
+    - expr_form: grain
     - kwarg:
-       channel: "#devops"
-       message: "Hubblestack Pulsar FMI - Detected change - Host:`{{ data['id'] }}` - File modified:`{{ data['path'] }}` - User:`{{ data['stats']['user'] }}`"
-       from_name: "saltbot"
-       api_key: {{ salt.pillar.get('slack_api_token') }}
\ No newline at end of file
+        channel: "#devops"
+        message: "Hubblestack Pulsar FMI - Detected change - Host:`{{ data['id'] }}` - File modified:`{{ data['path'] }}` - Details: `{{ data }}`"
+        from_name: "saltbot"
+        api_key: {{ salt.pillar.get('slack_api_token') }}
```

Comment:
Is this necessary still? Are we going to set the api_key in the minion pillar data?